extensions/blocks/contact-info/contact-info.php

```diff
@@ -10,31 +10,134 @@
 jetpack_register_block(
 	'jetpack/contact-info',
 	array(
-		'render_callback' => 'jetpack_contact_info_block_load_assets',
+		'render_callback' => array( 'Jetpack_Contact_Info_Block', 'render' )
 	)
 );
+
 jetpack_register_block(
-	'jetpack/email',
-	array( 'parent' => array( 'jetpack/contact-info' ) )
+	'jetpack/address',
+	array(
+		'parent' => array( 'jetpack/contact-info' ),
+	    'render_callback' => array( 'Jetpack_Contact_Info_Block', 'render_address' )
+	)
 );
+
 jetpack_register_block(
-	'jetpack/address',
-	array( 'parent' => array( 'jetpack/contact-info' ) )
+	'jetpack/email',
+	array(
+		'parent' => array( 'jetpack/contact-info' ),
+	    'render_callback' => array( 'Jetpack_Contact_Info_Block', 'render_email' )
+	)
 );
+
 jetpack_register_block(
 	'jetpack/phone',
-	array( 'parent' => array( 'jetpack/contact-info' ) )
+	array(
+		'parent' => array( 'jetpack/contact-info' ),
+		'render_callback' => array( 'Jetpack_Contact_Info_Block', 'render_phone' )
+	)
 );
 
 /**
- * Contact info block registration/dependency declaration.
+ * Class Jetpack_Contact_Info_Block
  *
- * @param array  $attr    Array containing the contact info block attributes.
- * @param string $content String containing the contact info block content.
+ * Helper class that lets us add schema attributes dynamically because they are not something that is store with the content
  *
- * @return string
  */
-function jetpack_contact_info_block_load_assets( $attr, $content ) {
-	Jetpack_Gutenberg::load_assets_as_required( 'contact-info' );
-	return $content;
+class Jetpack_Contact_Info_Block {
+
+	/**
+	 * Adds contact info schema attributes
+	 *
+	 * @param array  $attr    Array containing the contact info block attributes.
+	 * @param string $content String containing the contact info block content.
+	 *
+	 * @return string
+	 */
+	static function render( $attr, $content ) {
+		Jetpack_Gutenberg::load_assets_as_required( 'contact-info' );
+		if( $content )
+		return str_replace(
+			'class="wp-block-jetpack-contact-info"',
+			'class="wp-block-jetpack-contact-info" itemprop="location" itemscope itemtype="http://schema.org/Organization"',
+			$content );
+	}
+
+	/**
+	 * Adds address schema attributes
+	 *
+	 * @param array  $attr    Array containing the address block attributes.
+	 * @param string $content String containing the address block content.
+	 *
+	 * @return string
+	 */
+	static function render_address( $attr, $content ) {
+		if ( ! self::has_attributes( $attr, array( 'linkToGoogleMaps' ) ) ) {
```

Comment:
I think it may be useful to explain what this is and what it does in a comment above the check.